databricks/koalas/missing/series.py

```diff
@@ -87,15 +87,13 @@ class _MissingPandasLikeSeries(object):
     get_dtype_counts = unsupported_function('get_dtype_counts')
     get_ftype_counts = unsupported_function('get_ftype_counts')
     get_value = unsupported_function('get_value')
-    get_values = unsupported_function('get_values')
```

Comment:
need to undo this change

Reply:
Yes, you are right! 
I forget these changes in both `series.py` and `frame.py`.
@rxin  Thanks!